test/bdd/pkg/didexchange/didexchange_controller_steps.go

```diff
@@ -101,8 +101,14 @@ func (a *ControllerSteps) RegisterSteps(s *godog.Suite) { //nolint dupl
 }
 
 // EstablishConnection establishes connection between two agents through did exchange protocol
-func (a *ControllerSteps) EstablishConnection(inviter, invitee string) error {
-	return a.performDIDExchange(inviter, invitee)
+func (a *ControllerSteps) EstablishConnection(inviters, invitee string) error {
```

Comment:
Let's phase out did-exchange and instead add this in this out-of-band bdd package

Reply:
We do not have `outofband_controller_steps` tests for now. https://github.com/hyperledger/aries-framework-go/issues/1891